target/linux/brcm63xx/image/bcm63xx.mk

```diff
@@ -207,6 +207,19 @@ endef
 TARGET_DEVICES += R1000H
 
 ### ADB ###
+define Device/AGPWI
+  $(Device/bcm63xx)
+  DEVICE_VENDOR := ADB
+  DEVICE_MODEL := P.DG A4001N A-000-1A1-AX (AGPWI)
+  DEVICE_DTS := agpwi
+  CFE_BOARD_ID := 96328dg2x2
+  CFE_CHIP_ID := 6328
+  FLASH_MB := 8
+  DEVICE_PACKAGES := \
+    $(USB2_PACKAGES) $(B43_PACKAGES)
```

Comment:
Doesn't that fit into one line?

Reply:
If you refer to line 218 and 219 yes, but in the bcm63xx.mk file "$(USB2_PACKAGES) $(B43_PACKAGES)" is always in a new line after the "DEVICE_PACKAGES := \" for all the other boards.

I copied and paste these two lines from the ones in the A4001N without any modification to mantain the same code style.

Anyway if you want I put line 219 in the same line of 218

Reply:
I see your point. However, I do not see any advantage in a "two-line consistency" here, and would use one line anyway if it was my PR.
Since I did only a drive-by review though, and won't merge this, feel free to choose whatever you like more.